tests/schemas/api_v3/block.schema.js

```diff
@@ -34,7 +39,10 @@ const blockSchema = {
 	totalBurnt: Joi.string().required(),
 	maxHeightGenerated: Joi.number().required(),
 	maxHeightPrevoted: Joi.number().required(),
-	validatorsHash: Joi.string().required(),
+	validatorsHash: Joi.string().pattern(regex.HASH_SHA256).required(),
+	numberOfTransactions: Joi.number().integer().min(0).required(),
+	numberOfAssets: Joi.number().integer().min(1).required(),
+	numberOfEvents: Joi.number().integer().min(1).required(),
```

Comment:
Please confirm with the SDK team if there are scenarios where `numberOfAssets` and `numberOfEvents` could be `0`? If so, please adjust accordingly.